<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitterH">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitterV">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="ImageView" name="imageView"/>
       <widget class="PlotWidget" name="streamPlot"/>
      </widget>
      <widget class="ParameterTree" name="parameterTree"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoadStreamAs">
     <property name="title">
      <string>Load Stream(s) as...</string>
     </property>
     <addaction name="actionRefStream"/>
     <addaction name="actionTestStream"/>
    </widget>
    <addaction name="actionLoadCXI"/>
    <addaction name="actionLoadGeom"/>
    <addaction name="menuLoadStreamAs"/>
    <addaction name="separator"/>
    <addaction name="actionLoadState"/>
    <addaction name="actionSaveState"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowStreamTable"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadCXI">
   <property name="text">
    <string>Load CXI</string>
   </property>
   <property name="toolTip">
    <string>Load single cxi file</string>
   </property>
   <property name="statusTip">
    <string>Load single cxi file</string>
   </property>
  </action>
  <action name="actionLoadGeom">
   <property name="text">
    <string>Load Geometry</string>
   </property>
   <property name="toolTip">
    <string>Load geometry file (support CrystFEL, Cheetah and psana format)</string>
   </property>
   <property name="statusTip">
    <string>Load geometry file (support CrystFEL, Cheetah and psana format)</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShowStreamTable">
   <property name="text">
    <string>Show stream table</string>
   </property>
   <property name="statusTip">
    <string>Show stream table</string>
   </property>
  </action>
  <action name="actionRefStream">
   <property name="text">
    <string>reference stream</string>
   </property>
  </action>
  <action name="actionTestStream">
   <property name="text">
    <string>test stream(s)</string>
   </property>
  </action>
  <action name="actionTest_stream_2">
   <property name="text">
    <string>test stream 2</string>
   </property>
  </action>
  <action name="actionTest_stream_3">
   <property name="text">
    <string>test stream 3</string>
   </property>
  </action>
  <action name="actionLoadState">
   <property name="text">
    <string>Load State</string>
   </property>
  </action>
  <action name="actionSaveState">
   <property name="text">
    <string>Save State</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ParameterTree</class>
   <extends>QTreeWidget</extends>
   <header>pyqtgraph.parametertree</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
